client/root-error-boundary.tsx

```diff
@@ -9,7 +10,7 @@ import ResetStyle from './styles/reset'
 import { Headline5, subtitle1 } from './styles/typography'
 import { WindowControls, WindowControlsStyle } from './system-bar/window-controls'
 
-export interface RootErrorBoundaryProps {
+export interface RootErrorBoundaryProps extends WithTranslation {
```

Comment:
same thing as the other PR, you should put `WithTranslation` in the generic instead of adding it to this interface